### PR TITLE
fix(cxl-ui): always stick action bar to the bottom 

### PR DESCRIPTION
**Pull request by Elandriil**
https://app.clickup.com/t/ptfgc9

### CODE REVIEW

File: packages/cxl-ui/scss/cxl-app-layout/_layout.scss [hunk 1]
```diff
@@ -23,6 +23,11 @@ $toggle-icon: "lumo:angle-right";
 :host([layout^="2c"]) {
   main {
     > slot:not([name]) {
+      /**
+      * Force container height.
+      */
+      min-height: 90%;
```
Comment (Elandriil) [inline packages/cxl-ui/scss/cxl-app-layout/_layout.scss:29]: I used 90% here as 100 caused a scrollbar to appear.

Comment (lkraav) [reply packages/cxl-ui/scss/cxl-app-layout/_layout.scss:29]: 1. that's code-level comment material.
2. scrollbar === it's an incorrect solution, or some other layout element needs adjustment, too.

**Review by pawelkmpt — COMMENTED** [review]
I tested and it works, I'm just curious why there are `https://conversionxli.warmpress.com/` URLs in the story

EDIT
OK, it's a copy of `layout=2c-r.stories.js`

File: packages/cxl-ui/scss/cxl-app-layout/_layout.scss [hunk 2]
```diff
@@ -23,6 +23,15 @@ $toggle-icon: "lumo:angle-right";
...
   main {
     > slot:not([name]) {
+      /**
+      * Force container height.
+      *
+      * border-box is used to avoid overflow caused by padding.
+      * 60px is action bar height
+      */
+      box-sizing: border-box;
+      min-height: calc(100% - 60px);
```
Comment (lkraav) [inline packages/cxl-ui/scss/cxl-app-layout/_layout.scss:33]: How come I'm the only one fighting magic numbers @pawelkmpt?

This should be at least a CSS variable, not a hard-coded magic number.

File: packages/cxl-ui/scss/cxl-app-layout/_layout.scss [hunk 3]
```diff
@@ -23,6 +23,15 @@ $toggle-icon: "lumo:angle-right";
 :host([layout^="2c"]) {
   main {
     > slot:not([name]) {
+      /**
+      * Force container height.
+      *
+      * border-box is used to avoid overflow caused by padding.
+      * 60px is action bar height
+      */
+      box-sizing: border-box;
```
Comment (lkraav) [inline packages/cxl-ui/scss/cxl-app-layout/_layout.scss:32]: Now needing `border-box` signals this is probably not the right foundational solution, but lack of bandwidth means it'll just have to be improved later.

**Review by lkraav — COMMENTED** [review]
Could be better

File: packages/cxl-ui/scss/cxl-app-layout/_layout.scss (hunk 2 above)
Comment (pawelkmpt) [reply packages/cxl-ui/scss/cxl-app-layout/_layout.scss:33]: > How come I'm the only one fighting magic numbers @pawelkmpt?
> 
> This should be at least a CSS variable, not a hard-coded magic number.

I noticed, investigated, then tested and forgot about moving it to the var.

**Pull request merged**